spec.html

```diff
@@ -7536,6 +7535,18 @@ <h1>SetFunctionName ( _F_, _name_ [ , _prefix_ ] )</h1>
     </emu-clause>
 
     <!-- es6num="9.2.12" -->
+    <emu-clause id="sec-setfunctionlength" aoid="SetFunctionLength">
+      <h1>SetFunctionLength ( _F_, _length_)</h1>
+      <p>The abstract operation SetFunctionLength requires a Function argument _F_, and a Number argument _length_. This operation adds a `length` property to _F_ by performing the following steps:</p>
+      <emu-alg>
+        1. Assert: _F_ is an extensible object that does not have a `length` own property.
+        1. Assert: Type(_length_) is Number.
+        1. Assert: ToLength(_length_) is equal to _length_.
```

Comment:
Unsure if that's the best way to assert that `length` is an integer within the right bounds.